webofneeds/won-owner-webapp/src/main/webapp/app/components/account-menu.js

```diff
@@ -77,7 +77,11 @@ function genLogoutConf() {
     }
 
     getEmail() {
-      return ellipsizeString(this.email, this.maxEmailLength);
+      if (this.email.endsWith("matchat.org")) {
```

Comment:
i think we could use privateIdUser instead (is stored in the state)

Reply:
I looked into the state and tehre doesn't seem to be anything like that?

Reply:
nvm i just saw that i use the routeParam instead...e.g. topnav.js " isPrivateIdUser: !!privateId," should work too though not sure if i like it better than the email ending though...i guess noone will have a user with a matchat.org email anyway... maybe we just need to mark it with a todo for if we ever need to create privateIdUsers from a different domain though

Reply:
I extracted the call into a selector on the state. This should make it more reusable.